app/code/Magento/Sales/Model/Order.php

```diff
@@ -1831,7 +1847,7 @@ public function getCreatedAtFormatted($format)
             new \DateTime($this->getCreatedAt()),
             $format,
             $format,
-            null,
+            $this->localeResolver->getDefaultLocale(),
```

Comment:
Seems like this public method can be easily covered with unit test, however, as there is no logic and we just call method of another class with a bunch of arguments I do not insist.